sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/AssemblyReference.cs

```diff
@@ -48,6 +48,7 @@ internal static class MetadataReferenceHelper
             {
                 "lib",
                 "portable-net45",
+                "net20",
```

Comment:
Added net20 to the list of frameworks supported by the unit tests NuGet package installer (required since the legacy XUnit package is only net20).
The combination of LINQ and inner functions in this helper class made the debugging experience when I was trying to work out why the package test was failing significantly poorer than usual.